src/db/commit.rs

```diff
@@ -162,15 +198,16 @@ impl Commit {
     fn new_impl(
         mut builder: FlatBufferBuilder,
         local_create_date: &str,
-        basis_hash: Option<&str>,
+        basis_hash: Ref,
         checksum: Checksum,
         union_type: commit_fb::MetaTyped,
         union_value: flatbuffers::WIPOffset<flatbuffers::UnionWIPOffset>,
         value_hash: &str,
+        original_hash: Option<&str>,
```

Comment:
the original is in the meta if you want to pull it out of there instead of passing it in, which would be more DRY.

Reply:
It is in `(union_type, union_value)`. I'll see if I can extract the data from that.

Reply:
I could not figure out how to go back to a LocalMeta here